data/sql_updates/election_outcome.sql

```diff
@@ -1,45 +1,160 @@
--- See discussion in https://github.com/18F/openFEC-web-app/issues/514
+-- This view powers election search
 drop materialized view if exists ofec_election_result_mv_tmp;
 create materialized view ofec_election_result_mv_tmp as
-(
-    select fec_election_yr - 4 as election_yr, *
-    from disclosure.cand_valid_fec_yr
-    where
-        cand_ici = 'I' and
-        cand_office = 'P' and
-        fec_election_yr in (cand_election_yr, cand_election_yr + 4)
-    order by fec_election_yr desc
-)
-union all
-(
-    select distinct on (cand_office_st, cand_election_yr)
-        cand_election_yr - 6 as election_yr, *
+-- get the years with election records by office, state and district
+-- this data starts in 2003
+with election_dates_house_senate as(
+    select
+        election_state,
+        office_sought,
+        election_district,
+        election_yr
+    from fecapp.trc_election
+    where office_sought != 'P'
+        group by
+        election_state,
+        office_sought,
+        election_district,
+        election_yr
+),
+election_dates_president as(
+    select distinct on (election_yr)
+        'US'::char(2) as election_state,
```

Comment:
We just caught up with an in-person chat and this mirrors the output from before, so not only is it okay it's expected and other code is set to support it already.  We're good!